### ISSUE TITLE
Activity log cleanup

### ISSUE THREAD

**Issue by psouza4** [psouza4/mediacentermaster#1599]
Originally reported on Google Code with ID 1604

```
Describe the new feature or change:
Automatically delete logs older than 7 day (suggested).

Activity logs could be relatively small or rather large depending on your setup.  I
deleted 7 months worth of log the other day.  

List any references (websites or forum threads) that help describe this
request:
http://forums.mediacentermaster.com/viewtopic.php?f=5&t=9065&hilit=log+cleanup
```

Reported by `doug2h@hotmail.com` on 2014-11-19 20:05:37

**Comment by Happy2Play** [comment]
Any update on this?  Just deleted almost a years worth of logs.

**Comment by psouza4** [comment]
Added an option to automatically delete log files that have not been written to in the last 30 days.  The new option is enabled by default and runs as a background task in Media Center Master every hour.

Feature added and waiting for next release.

**Comment by psouza4** [comment]
Released update in version 2.17.26917.859.
